src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -103,27 +114,105 @@ function WorkspaceCategoriesPage({policyCategories, route}: WorkspaceCategoriesP
         </View>
     );
 
+    const navigateToCategorySettings = (category: PolicyOption) => {
+        Navigation.navigate(ROUTES.WORKSPACE_CATEGORY_SETTINGS.getRoute(route.params.policyID, category.text));
+    };
+
     const navigateToCategoriesSettings = () => {
         Navigation.navigate(ROUTES.WORKSPACE_CATEGORIES_SETTINGS.getRoute(route.params.policyID));
     };
 
-    const navigateToCategorySettings = (category: PolicyForList) => {
-        Navigation.navigate(ROUTES.WORKSPACE_CATEGORY_SETTINGS.getRoute(route.params.policyID, category.text));
+    const dismissError = (item: PolicyOption) => {
+        Policy.clearCategoryErrors(route.params.policyID, item.keyForList);
     };
 
-    const isLoading = !isOffline && policyCategories === undefined;
+    const selectedCategoriesArray = Object.keys(selectedCategories).filter((key) => selectedCategories[key]);
 
-    const settingsButton = (
-        <View style={[styles.w100, styles.flexRow, isSmallScreenWidth && styles.mb3]}>
-            <Button
-                medium
-                onPress={navigateToCategoriesSettings}
-                icon={Expensicons.Gear}
-                text={translate('common.settings')}
-                style={[isSmallScreenWidth && styles.w50]}
-            />
-        </View>
-    );
+    const getHeaderButtons = () => {
+        if (selectedCategoriesArray.length > 0) {
+            const isAllEnabled = selectedCategoriesArray.every((categoryName) => policyCategories?.[categoryName].enabled);
+
+            const options: Array<DropdownOption<DeepValueOf<typeof CONST.POLICY.CATEGORIES_BULK_ACTION_TYPES>>> = [
```

Comment:
Looks like this type - `DeepValueOf<typeof CONST.POLICY.CATEGORIES_BULK_ACTION_TYPES>` - is reused a few times. Does it make sense to extract it to a separate type?

Reply:
Yes, but I think we can also remove `DeepValueOf<typeof CONST.POLICY.CATEGORIES_BULK_ACTION_TYPES>` from <ButtonWithDropdownMenu>

Reply:
ButtonWithDropdownMenu accepts a generic type from options

Reply:
Sorry, I am not sure I understand your idea.

Reply:
check commit: 
[refactor(ts): remove generic from ButtonWithDropdownMenu](https://github.com/Expensify/App/pull/37734/commits/05ee7390fad6d65880f9947e01e89559d1d0853c)